addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/SqlInjectionScanRule.java

```diff
@@ -1852,13 +1853,20 @@ private void expressionBasedAttack(
         }
     }
 
+    // At this time the sqli tests just look for 0, 1, or anything in between, so the exact value
+    // here doesn't matter. Anything between 0 and 1 works.
+    private static final float HEURISTIC_WEIGHT = .99f;
+
```

Comment:
Move the comment before and drop the blank line

Reply:
Was that on a previous version? If not then I'm confused 😁

Reply:
Hmmm musta been 🤔